psamm/tests/test_yaml.py

```diff
@@ -114,6 +114,97 @@ def write_model_file(self, filename, contents):
             f.write(contents)
         return path
 
+    def test_long_string_model(self):
+        longString = '''---
```

Comment:
We don't run flake on the tests but the usual style should still be followed when possible. Specifically, variables should be `snake_case` not `camelCase`.